### DISCUSSION TITLE
An error occurred in trying to get the latest status details.

### DISCUSSION THREAD

**Discussion by bonelifer** [upptime/upptime#962] [Q&A]
A few problems. The README.md file shows my second site as down. It's not. It is behind cloudflare, so maybe that's the problem.  Next problem related to the title, is happening in Firefox, Chromium and Vivaldi. Page comes up flashes and produces this.

![error](https://github.com/upptime/upptime/assets/121736/a5641199-f4f1-485b-a6a2-3617ca2d63c9)

https://github.com/phpbbmodders/status
https://status.phpbbmodders.net/

**Comment by AnandChowdhary** [comment]
Can you please share any relevant console logs, maybe there's an error that stops us from accessing GitHub's API.

**Reply by AnandChowdhary** [reply]
![CleanShot 2024-05-21 at 18 43 44@2x](https://github.com/upptime/upptime/assets/2841780/9f714cfd-1156-4805-a948-3845d6f04871)

I am able to reproduce your issue. Looks like your GitHub repository is not public? If that's the case, you'll have to create a proxy to serve the API (https://upptime.js.org/docs/configuration/#custom-api-base-url) for the API to be available to the status site.

**Comment by bonelifer** [comment]
It's not private. ![notprivate](https://github.com/upptime/upptime/assets/121736/e342d3a9-3291-4544-a209-7fc28fad83fb)

**Comment by bonelifer** [comment]
From Setup CI: [logs_24012826792.zip](https://github.com/upptime/upptime/files/15402832/logs_24012826792.zip)

**Comment by bonelifer** [comment]
Since this is a organization public repo, I just changed it from my account bonelifer to the name of the organization phpbbmodders. Didn't help.

**Comment by bonelifer** [comment]
Uptime CI: [logs_24054141814.zip](https://github.com/upptime/upptime/files/15402914/logs_24054141814.zip)

Maybe I should delete and recreate the repo. Should the username be bonelifer or phpbbmodders?

**Reply by AnandChowdhary** [reply]
The username should be phpbbmodders.
